### PR TITLE
Add InfluxDB Enterprise file system layout

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/enterprise_influxdb/v1.9/concepts/file-system-layout.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+---
+
+The InfluxDB Enterprise file system layout depends on the installation method
+or containerization platform used to install InfluxDB Enterprise.
+
+- [InfluxDB Enterprise file structure](#influxdb-enterprise-file-structure)
+- [File system layout](#file-system-layout)
+
+## InfluxDB Enterprise file structure
+The InfluxDB file structure includes of the following:
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.9/concepts/file-system-layout.md:20]:
```suggestion
The InfluxDB file structure includes the following:
```

File: content/enterprise_influxdb/v1.9/concepts/file-system-layout.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+- [Data directory](#data-directory)
+- [WAL directory](#wal-directory)
+- [Metastore directory](#metastore-directory)
+- [Hinted handoff directory](#hinted-handoff-directory)
+- [Default InfluxDB Enterprise configuration files](#default-influxdb-enterprise-configuration-files)
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.9/concepts/file-system-layout.md:26]: Maybe...
```suggestion
- [InfluxDB Enterprise configuration files](#default-influxdb-enterprise-configuration-files)
```

**Review by pierwill — APPROVED** [review]
🎉

File: content/enterprise_influxdb/v1.9/concepts/file-system-layout.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+- [File system layout](#file-system-layout)
+
+## InfluxDB Enterprise file structure
+The InfluxDB file structure includes the following:
+
+- [Data directory](#data-directory)
+- [WAL directory](#wal-directory)
+- [Metastore directory](#metastore-directory)
+- [Hinted handoff directory](#hinted-handoff-directory)
+- [InfluxDB Enterprise configuration files](#default-influxdb-enterprise-configuration-files)
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.9/concepts/file-system-layout.md:26]:
```suggestion
- [InfluxDB Enterprise configuration files](#influxdb-enterprise-configuration-files)
```

File: content/enterprise_influxdb/v1.9/concepts/file-system-layout.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+Directory path of the InfluxDB Enterprise metastore, which stores information
+about the cluster, users, databases, retention policies, shards, and continuous queries.
+
+**On data nodes**, the metastore contains information about InfluxDB Enterprise meta nodes.
+To customize this path, use the [`[meta].dir` configuration option in your data node configuration file](/enterprise_influxdb/v1.9/administration/config-data-nodes/#dir--varlibinfluxdbmeta).
+
+**On meta nodes**, the metastore contains information about the InfluxDB Enterprise RAFT cluster.
+To customize this path, use the [`[meta].dir` configuration option in your meta node configuration file](/enterprise_influxdb/v1.9/administration/config-meta-nodes/#dir--varlibinfluxdbmeta).
+
+### Default configuration files
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.9/concepts/file-system-layout.md:56]:
```suggestion
### InfluxDB Enterprise configuration files
```

File: content/enterprise_influxdb/v1.9/concepts/file-system-layout.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+- [Meta node file system layout](#meta-node-file-system-layout)
+
+### Data node file system layout
+| Path                                                     | Default                       |
+| :------------------------------------------------------- | :---------------------------- |
+| [Data directory](#data-directory)                        | `/var/lib/influxdb/data/`     |
+| [WAL directory](#wal-directory)                          | `/var/lib/influxdb/wal/`      |
+| [Metastore directory](#metastore-directory)              | `/var/lib/influxdb/meta/`     |
+| [Hinted handoff directory](#hinted-handoff-directory)    | `/var/lib/influxdb/hh/`       |
+| [Default config file path](#default-configuration-files) | `/etc/influxdb/influxdb.conf` |
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.9/concepts/file-system-layout.md:77]:
```suggestion
| [Default config file path](#influxdb-enterprise-configuration-files) | `/etc/influxdb/influxdb.conf` |
```

File: content/enterprise_influxdb/v1.9/concepts/file-system-layout.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+    - client.json
+  - wal/
+    - _<span style="opacity:.4">WAL directories and files</span>_
+{{% /filesystem-diagram %}}
+
+### Meta node file system layout
+| Path                                                     | Default                            |
+| :------------------------------------------------------- | :--------------------------------- |
+| [Metastore directory](#metastore-directory)              | `/var/lib/influxdb/meta/`          |
+| [Default config file path](#default-configuration-files) | `/etc/influxdb/influxdb-meta.conf` |
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.9/concepts/file-system-layout.md:98]:
```suggestion
| [Default config file path](#influxdb-enterprise-configuration-files) | `/etc/influxdb/influxdb-meta.conf` |
```

**Pull request merged**